common/utils/utils.go

```diff
@@ -289,6 +294,12 @@ func ContainsAny(s string, l []string) bool {
 	return false
 }
 
+func ValueSetEqual(a, b []string) bool {
```

Comment:
the name is unclear.
You can rename it to something like checkEqualSets or something